docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4503,6 +4503,9 @@ List tasks
   - `id=<task id>`
   - `name=<task name>`
   - `service=<service name>`
+  - `node=<node id>`
+  - `label=key` or `label="key=value"`
+  - `desired-state=(running | shutdown | accepted)>`
```

Comment:
can you change this to

    `desired-state=<running | shutdown | accepted>`

(pointy brackets)
to illustrate the value is required